File: 05. Percabangan/D. If Then Else/d_if_then_else.c
#include <stdio.h>
int main()
{
    int N;
    scanf("%d", &N);
    if (N > 0){
        printf("positif\n");
    }
    else if (N < 0){
        printf("negatif\n");
    }
    else{
        printf("nol\n");
    }
    return 0;
}

File: 05. Percabangan/D. If Then Else/d_if_then_else.go
package main

import (
	"fmt"
)

func main() {
	var N int
	fmt.Scanf("%d", &N)
	if N > 0 {
		fmt.Println("positif")
	} else if N < 0 {
		fmt.Println("negatif")
	} else {
		fmt.Println("nol")
	}
}
